packages/backend/src/core/transaction-count/RpcTransactionUpdater.ts

```diff
@@ -61,6 +61,19 @@ export class RpcTransactionUpdater implements TransactionCounter {
     })
   }
 
+  // We need to subtract the one system transaction that appear in each Nitro block
+  private calcTransactionCount(blockNumber: number, transactions: string[]) {
```

Comment:
It would be cleaner if this function was configurable through constructor instead of hardcoded project ids.

Reply:
Wanted to do it like this from the beginning, thanks for help!